store/postgres/src/filter.rs

```diff
@@ -426,5 +439,49 @@ where
                 }
             }
         }
+
+        Intersects(attribute, values) => {
+            fn pred<QS, T>(
+                attribute: String,
+                values: Vec<Value>,
+                coercion: &str,
+            ) -> Result<FilterExpression<QS>, UnsupportedFilter>
+            where
+                QS: EntitySource + 'static,
+                Pg: HasSqlType<T>,
+                T: 'static,
+                SqlValue: ToSql<T, Pg>,
+            {
+                // There's no good way to check whether two JSONB arrays
+                // intersect. We do it by turning the `data` parts of the
+                // JSONB array entries into a table of strings and then
+                // check if any of them appear in `values`. This will be slow
+                // if the array in `attribute` is large
+                Ok(Box::new(
+                    sql::<Bool>("exists (select 1 from jsonb_array_elements(data->")
+                        .bind::<Text, _>(attribute)
+                        .sql("->'data') d where (d->>'data')")
+                        .sql(coercion)
+                        .sql(" = any(")
+                        .bind::<Array<T>, _>(SqlValue::new_array(values))
+                        .sql("))"),
```

Comment:
Do we use this only for JSONB schemas or for relational schemas as well?

Reply:
Everything in this file is only used for JSONB storage; the corresponding code for relational storage is [here](https://github.com/graphprotocol/graph-node/blob/master/store/postgres/src/relational_queries.rs#L382-L741)